core/src/main/java/org/infinispan/xsite/XSiteAdminCommand.java

```diff
@@ -104,8 +104,8 @@ public void writeTo(ObjectOutput output) throws IOException {
             return;
          case AMEND_TAKE_OFFLINE:
             output.writeUTF(siteName);
-            output.writeInt(afterFailures);
-            output.writeLong(minTimeToWait);
+            output.writeObject(afterFailures);
+            output.writeObject(minTimeToWait);
```

Comment:
I don't see why. why waste memory with fields that aren't going to be used? you would have to allocate larger buffers and use more bandwidth...